packages/dev/create-react-app/templates/default/src/parcel/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import logoUrl from 'url:./logo.svg';
+import frontUrl from 'url:./front.png';
+import backUrl from 'url:./back.png';
+import './index.css';
+
+export default function Parcel() {
+  return (
+    <div className="parcel-container">
+      <div className="parcel-heading">
+        <div>
+          <h1>Welcome to</h1>
+          <img src={logoUrl} className="parcel-logo" alt="Parcel logo" />
+        </div>
+
+        <div className="parcel-box">
+          <img src={backUrl} alt="" height="474" />
+          <div id="icons" />
```

Comment:
Was thinking of bringing over the animated icons from https://parceljs.org/, thus the split back and front images. However, we should take care to make all of this initial content easily removable in favor of the user's app content. Right now, that animation includes many svgs for the different file types shown in that animation.

We could combine the svgs together into a single sheet and use `use` to reference each type, and inline that svg sheet, the above stylesheet, and `icons.js` into this component to keep this to only a few files.